graylog2-web-interface/src/components/configurations/UrlWhiteListForm.jsx

```diff
@@ -0,0 +1,213 @@
+    } catch (e) {
+      isValid = false;
+    }
+    return isValid;
+  };
+
+  const _isFormValid = (): boolean => {
+    let isValid = true;
+    if (validationState.errors.length > 0
+      && validationState.errors.find((el => (el && el.title && el.title.valid) === false
+      || (el && el.value && el.value.valid === false)))) {
+      isValid = false;
+    }
+    return isValid;
+  };
+
+  const _updateState = (idx: number, type: string, name: string, value: string) => {
+    const stateUpdate = ObjectUtils.clone(config);
+    stateUpdate.entries[idx][name] = value;
+    stateUpdate.entries[idx] = { ...stateUpdate.entries[idx], type };
+    setConfig(stateUpdate);
+  };
+
+  const _updateValidationError = (idx: number, type: string, name: string, result: Object, value: string) => {
+    const validationUpdate = ObjectUtils.clone(validationState);
+    validationUpdate.errors[idx] = { ...validationUpdate.errors[idx], [name]: result };
+    setValidationState(validationUpdate);
+    _updateState(idx, type, name, value);
+  };
+
+  const _validate = (name: string, idx: number, type: string, value: string): void => {
+    switch (name) {
+      case 'title': {
+        const result = value.trim().length <= 0 ? { valid: false } : { valid: true };
+        _updateValidationError(idx, type, name, result, value);
+      }
+        break;
+      case 'value':
+        if (type === literal) {
+          const result = validURL(value) ? { valid: true } : { valid: false };
+          _updateValidationError(idx, type, name, result, value);
+        } else if (type === regex && value.trim().length > 0) {
+          const promise = ToolsStore.testRegexValidity(value);
+          promise.then((result) => {
+            const res = result.is_valid ? { valid: true } : { valid: false };
+            _updateValidationError(idx, type, name, res, value);
+          });
+        } else {
+          const res = { valid: false };
+          _updateValidationError(idx, type, name, res, value);
+        }
+        break;
+      default:
+        break;
+    }
+  };
+
+  const _onInputChange = (event: SyntheticInputEvent<EventTarget>, idx: number, type: string) => {
+    _validate(event.target.name, idx, type, FormUtils.getValueFromInput(event.target));
+  };
+
+  const _onUpdateType = (idx: number, type: string) => {
+    const stateUpdate = ObjectUtils.clone(config);
+    _validate('value', idx, type, stateUpdate.entries[idx].value);
+  };
+  const _getSummary = () => {
+    return (config.entries.map((url, idx) => {
+      return (
+      // eslint-disable-next-line react/no-array-index-key
+        <tr key={url.id}>
+          <td>{idx + 1}</td>
+          <td>
+            <Input type="text"
+                   id={`title-input${idx}`}
+                   ref={(elem) => { inputs[`title${idx}`] = elem; }}
+                   help={validationState.errors[idx] && validationState.errors[idx].title && !validationState.errors[idx].title.valid ? 'required' : null}
```

Comment:
Here I think we should also be more verbose and make it consistent with the other field, maybe write something like: `Required field`.

Reply:
fixed

Reply:
done